packages/dev/docs/pages/blog/creating-a-pointer-friendly-submenu-experience.mdx

```diff
@@ -0,0 +1,119 @@
+{/* Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import SubmenuSafeArea from '../assets/submenu-safe-area.svg';
+import SubmenuAtan2 from '../assets/submenu-atan2.svg';
+import SubmenuAtan2FromPointer from '../assets/submenu-atan2-from-pointer.svg';
+
+import {BlogPostLayout} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+keywords: [react aria, react spectrum, react, spectrum, interactions, submenu, pointer]
+description: We are excited to announce support for submenus in the latest release of React Spectrum, React Aria, and React Aria Components! In the process of adding this feature, we found ourselves solving some unique challenges while working to make submenus user-friendly and accessible across an array of devices and input types. In doing so, we wanted to share our thought process in solving one of the challenges we faced along the way.
+date: 2024-04-03
+author: '[Reid Barber](https://github.com/reidbarber)'
+image: ../assets/submenu-safe-area.png
+---
+
+# Creating a pointer-friendly submenu experience
+
+We are excited to announce a beta release of submenus in the latest release of React Spectrum, React Aria, and React Aria Components! In the process of adding this feature, we found ourselves solving some unique challenges while working to make submenus user-friendly and accessible across an array of devices and input types. In doing so, we wanted to share our thought process in solving one of the challenges we faced along the way.
+
+## The Shortcut Problem
```

Comment:
Why is it called the "shortcut" problem? You didn't mention shortcut again so I wasn't sure.

Reply:
I made that up. Wasn't sure of a better section heading. I'll update that section to reference it.

Reply:
Renamed this to `The Shortest Path` to make it more clear.